makelib/uptest.mk

```diff
@@ -54,6 +54,13 @@ uptest: $(UPTEST) $(KUBECTL) $(CHAINSAW) $(CROSSPLANE_CLI) $(YQ)
 # Run uptest together with all dependencies. Use `make e2e UPTEST_SKIP_DELET=true` to skip deletion of resources.
 e2e: build controlplane.down controlplane.up $(UPTEST_LOCAL_DEPLOY_TARGET) uptest #
 
+ifdef LANG_KCL
```

Comment:
where are we documenting this convention? How is it exposed to the user?

Reply:
for now it's just a proposal, if you're fine with the mechanism i'd add some docs here and in project-makefiles :)